tiledb/sm/storage_manager/context_resources.cc

```diff
@@ -31,28 +31,48 @@
  */
 
 #include "tiledb/sm/storage_manager/context_resources.h"
+#include "tiledb/sm/rest/rest_client.h"
 
 using namespace tiledb::common;
 
 namespace tiledb::sm {
 
+#ifdef TILEDB_SERIALIZATION
+constexpr bool TILEDB_SERIALIZATION_ENABLED = true;
```

Comment:
Only thing I don't care for in this PR is this definition. Should probably be in a namespace such as `tdb::project_config` and be defined in somewhere that both the library and tests for it can access it. And lower case like other variables. It's only used in one place right now, so it would be easy to rewrite soon afterwards.

Reply:
Lets do the project_config change separately. There’s roughly 250 places we’ll need to change with if constexpr statements around who knows how many lines of code. Which is to say I expect that PR to be fairly large by lines of code changed count.

Reply:
Story link for the project_config work: https://app.shortcut.com/tiledb-inc/story/25192/create-new-project-config-namespace-to-hold-bootstrap-options